exercises/exercise 1/README.md

```diff
@@ -28,6 +28,8 @@ https://hub.docker.com
 This workshop uses [Play With Docker](https://labs.play-with-docker.com), so everyone can have the same experience and we don't rely on WiFi to push/pull Gigabytes of images.
 You can still follow the instructions locally with your own engine.
 
+**Don't forget to save on your laptop all your exercises before leaving the workshop**.
```

Comment:
`You may want to set a reminder to save your exercises locally before leaving the workshop`?

Reply:
Hope we won't forget to remind them to save 😉